### ISSUE TITLE
By clicking on the mentioned name nothing happens 

### ISSUE THREAD

**Issue by Maria-kamenskaya** [nextcloud/talk-ios#551]
**Is your feature request related to a problem? Please describe.**
If I go to the chat and see someone’s name mentioned, for example “thank you @Ivan”, I click on it and nothing happens. It is recognized as a normal text, nothing more.

**Describe the solution you'd suggest**
By clicking on the mentioned name the messages of that person could be highlighted and you could be navigated to them if you tap on the right (there will be a button which shows how many messages this person sent and which of them is highlighted right now, like “1/25” button on the right side.

**Describe alternatives you've considered**
It could be also a helpful improvement if by clicking on the mentioned name you could be navigated to a last message of this person which will be highlighted (maybe as a short Circle animation).

**Additional context**

**Comment by jancborchardt** [comment]
cc @Ivansss ^ :)

**Comment by Ivansss** [comment]
Not sure about this one :/

I would expect some actions related with that user when clicking on top of some other user mention, like:
- Send a private message to that user
- Mention that user
- ...

When clicking on own mentions, it might be useful to have a feature like the one you described.
Having a counter with some arrows to navigate between other mentions to you in the current conversation.

What do you think @Maria-kamenskaya @jancborchardt ? :)

**Comment by jancborchardt** [comment]
Good point @ivansss – so @Maria-kamenskaya it sounds like a big issue is that the flow of a specific thread or conversation in a room could be clearer. I find it interesting how iMessage does it where you can tap on a quote and it can show the thread:
![](https://www.apple.com/v/ios/ios-14/c/images/overview/messages/inline_replies__gkd53a5ar7e6_large.jpg)

More details at
- https://www.apple.com/ios/ios-14/
- https://techcrunch.com/2020/06/22/apples-imessage-update-takes-cues-from-slack-with-mentions-pins-threads-and-more
